src/main/scala/org/apache/pekko/persistence/dynamodb/journal/DynamoDBHelper.scala

```diff
@@ -32,6 +32,35 @@ import scala.concurrent.duration._
 case class LatencyReport(nanos: Long, retries: Int)
 private class RetryStateHolder(var retries: Int = 10, var backoff: FiniteDuration = 1.millis)
 
+/**
+ * Auxiliary object to help determining whether we should retry on a certain throwable.
+ */
+object DynamoRetriableException {
```

Comment:
This should be marked `@InternalApi` and also be made package private. I don't think there is any usecase where the user would want to override this and if so then the structure should be redesigned to better accomodate that.

Reply:
Done